src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -356,6 +367,8 @@ export default class AddStaffNotationDialog extends Vue {
           fileType = 'put back on filing'
         } else if (this.isAdministrativeDissolution) {
           fileType = 'administrative dissolution filing'
+        } else if (this.isFilingTypeAdminFreeze) {
```

Comment:
Which code block builds the filing body for this new filing? If it is the block on lines 348-355 then please update the comment there.

Reply:
Also, I'm thinking there's a property missing in the filing to indicate whether it is Freezing or Unfreezing the business.